static/js/components/ProgramSelector_test.js

```diff
@@ -78,7 +75,9 @@ describe('ProgramSelector', () => {
   });
 
   it("does not render the 'Enroll in a new program' option if there is not at least one available program", () => {
-    let allEnrollments = enrollments.concat(unenrolled);
+    let allEnrollments = enrollments.map(program => Object.assign({}, program, {
```

Comment:
here's a line (along with line 58) that kinda illustrates my point about the naming. the collection is called `enrollments`, but the individual members are called `program`? i think the instinct is right here; the collection should just be called `programs`